src/gui/gtkentry.c

```diff
@@ -179,6 +179,7 @@ const dt_gtkentry_completion_spec *dt_gtkentry_get_default_path_compl_list()
           { "VERSION.NAME", N_("$(VERSION.NAME) - version name from metadata") },
           { "JOBCODE", N_("$(JOBCODE) - job code for import") },
           { "SEQUENCE", N_("$(SEQUENCE) - sequence number") },
+          { "SEQUENCE[4,1]", N_("$(SEQUENCE[n,m]) - sequence number, n: number of digits, m: start number") },
```

Comment:
We document only this, in the doc we should say that the parameters are optional.

Reply:
witg doc you mean not here but in the full documentation, right?

Reply:
Right!